manubot/cite/curie/__init__.py

```diff
@@ -67,16 +76,13 @@
 @dataclasses.dataclass
 class Handler_CURIE(Handler):
     def __post_init__(self):
-        try:
-            self.resource = get_prefix_to_resource()[self.prefix_lower]
-        except KeyError:
+        self.resource = manager.get_resource(self.prefix_lower)
+        if self.resource is None:
             raise ValueError(f"Unrecognized CURIE prefix {self.prefix_lower}")
-        self.standard_prefix = (
-            self.resource.get("preferred_prefix") or self.resource["prefix"]
```

Comment:
Shouldn't we still do `self.standard_prefix = self.resource.get_preferred_prefix()` here, since we want the standard prefix to have the preferred capitalization?